src/Dev/Tasks/TagsToShortcodeHelper.php

```diff
@@ -232,7 +232,7 @@ private function getTagTuple($tag)
             $this->validTagsPattern,
             $this->validAttributesPattern
         );
-        preg_match($pattern, $tag, $matches);
+        preg_match($pattern ?? '', $tag ?? '', $matches);
```

Comment:
```suggestion
        preg_match($pattern ?? '//', $tag ?? '', $matches);
```

Reply:
May be worth looking at where `$pattern` is assigned since this needs to be touched anyway and seeing if that null coalescing operator is needed